svir/test/unit/test_loss_aggreagation.py

```diff
@@ -46,7 +46,7 @@ def setUp(self):
         curr_dir_name = os.path.dirname(__file__)
         self.data_dir_name = os.path.join(
             curr_dir_name, os.pardir, 'data', 'aggregation', 'dummy')
-        self.loss_attr_names = ['FATALITIES', 'STRUCTURAL']
+        self.loss_attr_names = ['PEOPLE', 'STRUCTURAL']
```

Comment:
I see a confusion between loss type and unit of measure but not an easy solution (i.e. instead of PEOPLE we should have FATALITIES, INJURED or HOMELESS but then you would need to transmit the information from the engine to know what to use, while PEOPLE is good for everything).

Reply:
Anyway, this unit test is just checking that the plugin correctly aggregates "geographically" the values of a layer of points containing values of any kind, with respect to another layer containing polygons. I made that unneeded change only for the sake of naming consistency, but I could be aggregating anything.